### PR TITLE
terraform/cloudflare: add nix-community.org verification for github

### PR DESCRIPTION
**Pull request by zowoq**
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages#verifying-a-domain-for-your-organization-site

### CODE REVIEW

File: terraform/cloudflare_nix-community_org.tf [hunk 1]
```diff
@@ -103,6 +103,13 @@ resource "cloudflare_record" "nix-community-org-apex-TXT" {
   type    = "TXT"
 }
 
+resource "cloudflare_record" "nix-community-org-github-pages-challenge" {
+  zone_id = local.nix_community_org_zone_id
+  name    = "?"
```
Comment (Mic92) [inline terraform/cloudflare_nix-community_org.tf:108]:
```suggestion
  name    = "_github-pages-challenge-nix-community"
```

File: terraform/cloudflare_nix-community_org.tf [hunk 2]
```diff
@@ -103,6 +103,13 @@ resource "cloudflare_record" "nix-community-org-apex-TXT" {
...
+  value   = "?"
```
Comment (Mic92) [inline terraform/cloudflare_nix-community_org.tf:109]:
```suggestion
  value   = "6d236784300b9b1e80fdc496b7bfce"
```

Comment (zowoq) [reply terraform/cloudflare_nix-community_org.tf]: Thanks.

**Comment by Mic92** [comment]
bors merge

**Pull request merged**